### PR TITLE
Configure deploy provider settings

### PR DESCRIPTION
**Pull request by liuhaoyang**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -22,4 +22,13 @@ build_script:
 - ps: ./build.ps1
 test: off
 artifacts:
-- path: artifacts/packages/*.nupkg
\ No newline at end of file
+- path: artifacts/packages/*.nupkg
+deploy:
+- provider: NuGet
+  server: https://www.myget.org/F/skywalking/api/v2/package
```
Comment (wu-sheng) [inline appveyor.yml]: Why named under `v2`? Which version is this about?

File: appveyor.yml [hunk 2]
```diff
@@ -22,4 +22,13 @@ build_script:
...
+  api_key:
+    secure: y93uJrdiEYaeVTFxeCduQSlCussOUF4UztOyhkxyZxFnIco73J/K2rdA2QShoEuh
```
Comment (wu-sheng) [inline appveyor.yml]: Secure key is safe enough here?

File: appveyor.yml (hunk 1 above)
Comment (liuhaoyang) [reply appveyor.yml]: This is the push server, which you can see in myget's feed details

File: appveyor.yml (hunk 2 above)
Comment (liuhaoyang) [reply appveyor.yml]: Sure. It is encrypted using [appveyor encrypt-tool](https://ci.appveyor.com/tools/encrypt)

Comment (wu-sheng) [reply appveyor.yml]: If it has been encrypted by CI platform, then good enough for me.

**Comment by wu-sheng** [comment]
Do you intend to update your commit logs?

**Comment by liuhaoyang** [comment]
Thanks. You can merge this pull request, it doesn't matter to me. This seems to be a problem with the Git config in my pc,and I will fix it in later.

**Pull request merged**